weco_datascience/http.py

```diff
@@ -13,9 +13,9 @@ def start_persistent_client_session():
     _session_store["session"] = aiohttp.ClientSession()
 
 
-def close_persistent_client_session():
+async def close_persistent_client_session():
```

Comment:
Ha, I'd been wondering why this errors but hadn't cared since it happens when the app closes